main/api/dependencies/auth.py

```diff
@@ -0,0 +1,66 @@
+from datetime import datetime, timedelta
```

Comment:
Please only stored dependencies here, those authentication-related helper methods can move to the `auth service`